proposals/lip-0041.md

```diff
@@ -350,8 +351,8 @@ def execute(trs: Transaction) -> None:
     for signature, key in zip(trs.params.signatures, allKeys):
         if not verifyMessageSig(key, MESSAGE_TAG_MULTISIG_REG, networkIdentifier, message, signature):
             emitPersistentEvent(
-                moduleID = MODULE_ID_AUTH,
-                typeID = TYPE_ID_INVALID_SIGNATURE_ERROR,
+                module = MODULE_NAME_AUTH,
```

Comment:
Comment for the first line of this function: Reminder that Transaction type is not defined here. Not sure we should add it everywhere, but I am adding it in my LIPs so I guess it is good to do it for consistency.

Reply:
I think we decided not to define it and assume it is known.